docs/use-linea/fund.md

```diff
@@ -4,16 +4,16 @@ description: Use a faucet to obtain test ETH and USDC
 sidebar_position: 3
 ---
 
-Use [MetaMask](https://docs.metamask.io/guide/getting-started.html) to store your funds and interact with the ConsenSys zkEVM network. You can [install MetaMask as a browser extension or mobile app](https://metamask.io/download/).
+Use [MetaMask](https://docs.metamask.io/guide/getting-started.html) to store your funds and interact with Linea. You can [install MetaMask as a browser extension or mobile app](https://metamask.io/download/).
 
 ## Get test ETH on Goerli
 
 Use a faucet to obtain Goerli ETH. Transactions are much cheaper than Mainnet. Therefore 0.1 ETH is enough to execute a basic workflow. You can [view a list of faucets](https://faucetlink.to/goerli) for the Goerli network.
 
 ## Get test USDC on Goerli
 
-To obtain test USDC on Goerli, use our [USDC faucet](https://bridge.goerli.zkevm.consensys.net/faucet?token=USDC) to mint 10 USDC. You must have some Goerli test ETH to pay for the transaction to mint USDC.
+To obtain test USDC on Goerli, use our [USDC faucet](https://bridge.goerli.linea.build/faucet?token=USDC) to mint 10 USDC. You must have some Goerli test ETH to pay for the transaction to mint USDC.
```

Comment:
The bridge domain has not been migrated over yet. I brought it up earlier today in Slack https://consensys.slack.com/archives/C04ATK1LXDM/p1679679985475689. 

CC @julien-marchand

Reply:
Ah yeah I was wondering if it would be migrated by the due date

Reply:
If not I can roll back this change